control/veda_deliver_3play.py

```diff
@@ -105,11 +105,16 @@ def get_available_languages(self):
         """
         Gets all the 3Play Media supported languages
         """
-        response = requests.get(url=build_url(self.base_url, self.available_languages_url, apikey=self.api_key))
+        available_languages_url = build_url(self.base_url, self.available_languages_url, apikey=self.api_key)
+        response = requests.get(
+            url=available_languages_url
+        )
         if not response.ok:
             raise ThreePlayMediaLanguagesRetrievalError(
-                'Error while retrieving available languages: {response} -- {status}'.format(
-                    response=response.text, status=response.status_code
+                'Error while retrieving available languages: url={url} -- {response} -- {status}'.format(
+                    url=scrub_url_info(available_languages_url, ['apikey']),
```

Comment:
missed out L#167

Reply:
Both the requests are post requests and we have payload available so we can directly log the interested params.

Reply:
Ops right!